session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -525,16 +584,12 @@ public SessionDataSet executeQueryStatement(String sql, long timeoutInMs)
   private SessionDataSet executeStatementMayRedirect(String sql, long timeoutInMs)
       throws StatementExecutionException, IoTDBConnectionException {
     try {
-      logger.info("{} execute sql {}", defaultSessionConnection.getEndPoint(), sql);
+      logger.info("{} execute sql {}", defaultEndPoint, sql);
```

Comment:
`defaultSessionConnection.getEndPoint() `may not equal `defaultEndPoint` as long as we enable redirection